seatunnel-translation/seatunnel-translation-flink/seatunnel-translation-flink-13/src/main/java/org/apache/seatunnel/translation/flink/sink/FlinkSinkWriterContext.java

```diff
@@ -50,6 +50,11 @@ public int getIndexOfSubtask() {
         return writerContext.getSubtaskId();
     }
 
+    @Override
+    public int getNumberOfParallelSubtasks() {
```

Comment:
Please do not return 0 directly, if this is not necessary to implement, please give a default implementation in the interface, do not implement it here.